filters/reprojection/ReprojectionFilter.cpp

```diff
@@ -125,6 +125,8 @@ void ReprojectionFilter::initialize()
             "for the 'out_srs' option.";
         throw pdal_error(oss.str());
     }
+
+    if (!m_inferInputSRS) createTransform(nullptr);
```

Comment:
No one-line if/then's please and `nullptr` shouldn't be used as it isn't supported on some platforms.